frontend/src/components/Checkbox/index.module.css

```diff
@@ -0,0 +1,33 @@
+.label {
+  color: var(--color-black);
+  pointer-events: none;
```

Comment:
What does this do?

Reply:
That should've been color purple but now that I see it doesn't even do anything so I'll rethink all of this. The part of pointer-events was because I kept pressing the label and it checked and unchecked the box and I thought that was odd.

Reply:
Turns out I'm an airhead and I forgot putting the className on <label> :)

Reply:
The label pressing thing is a [feature](https://css-tricks.com/html-inputs-and-labels-a-love-story/#:~:text=There%20are%20two%20ways%20to,input%20and%20holding%20its%20hand.) not a bug.. keep it! (For your info, it's achieved by the `for` and `id`.